tests/Functional/TerminusTestBase.php

```diff
@@ -38,8 +38,21 @@ protected static function callTerminus(
         string $command,
         ?string $pipeInput = null
     ): array {
-        $terminusHost = isset($_ENV['TERMINUS_HOST']) ? "TERMINUS_HOST=" . $_ENV['TERMINUS_HOST'] : "";
```

Comment:
this looks like it was pulling the environment variable... was this not working correctly or something?

Reply:
n/m, i think i see... this was only handling 1 variable but you need 5...

Reply:
Because it's run in a separate process, that process has almost none of the environment variables of the parent process. It has to take everything it needs into the execution proc.

Reply:
Alright, well if this doesn't work for some reason, note that `proc_open` (called on line 60) takes a 5th argument for env vars: https://www.php.net/manual/en/function.proc-open